.github/actions/build-extension-package/docker/build_ubuntu_deb.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
similar to https://github.com/postgresml/postgresml/blob/master/pgml-extension/docker/build_ubuntu_deb.sh